### PR TITLE
Multiple attempts footer

### PR DESCRIPTION
**Pull request by jivey**
image

### CODE REVIEW

File: tutor/src/screens/task/step/exercise-question.js [hunk 1]
```diff
@@ -181,6 +139,25 @@ export default class ExerciseQuestion extends React.Component {
         );
     }
 
+    renderPoints(step) {
+        const points = [step.available_points];
+
+        if (!isNil(step.published_points_without_lateness)) {
+            points.unshift(step.published_points);
```
Comment (Dantemss) [inline tutor/src/screens/task/step/exercise-question.js:146]: It is a little odd to me that published_points_without_lateness is guarded here when it is published_points that is added, but I suppose they are equivalent in this regard.

File: tutor/src/components/buttons/save-practice.js [hunk 2]
```diff
@@ -81,7 +82,7 @@ const SavePracticeButton = observer(({
             practiceQuestion.destroy();
         }
         else {
-            practiceQuestions.create(taskStep.tasked_id);
+            runInAction(() => practiceQuestions.create(taskStep.tasked_id));
```
Comment (Dantemss) [inline tutor/src/components/buttons/save-practice.js:85]: Wouldn't destroy above also need this runInAction()?

**Review by Dantemss — APPROVED** [review]

File: tutor/src/screens/task/step/exercise-question.js (hunk 1 above)
Comment (jivey) [reply tutor/src/screens/task/step/exercise-question.js:146]: I thought the same thing, it was also there in the old code, so I assumed it had a reason. Meant to ask you about it 😄

File: tutor/src/components/buttons/save-practice.js (hunk 2 above)
Comment (jivey) [reply tutor/src/components/buttons/save-practice.js:85]: I almost added it too but then saw that `destroy` here is a wrapper that calls the actual `map.delete` (with `runInAction`) after doing the network request.

**Pull request merged**